src/scripts/components/story/story.tsx

```diff
@@ -36,16 +31,21 @@ const Story: FunctionComponent = () => {
     currentStoryId && dispatch(fetchStory(currentStoryId));
   }, [dispatch, currentStoryId]);
 
+  const storyList = useSelector(storyListSelector);
+  const currentStory = storyList.find(story => story.id === currentStoryId);
```

Comment:
Hmm now we have `currentStory` and `selectedStory`. Can we move these two lines also in the storyParams hook and just return a new prop storyListItem?